functional/FunctionalTest.java

```diff
@@ -2799,7 +2789,17 @@ public static void setBucketLifecycle() throws Exception {
     try {
       client.makeBucket(MakeBucketArgs.builder().bucket(bucketName).build());
       try {
-        testSetBucketLifecycle(bucketName);
+        testSetBucketLifecycle(
+            bucketName,
+            new LifecycleRule(
+                Status.ENABLED,
+                null,
+                new Expiration((ZonedDateTime) null, 365, null),
+                new RuleFilter("logs/"),
+                "rule2",
+                null,
+                null,
+                null));
```

Comment:
Since the exact same code is being used at multiple places to create this lifecycle rule, maybe it can be defined as a method or a static final class level variable?

Reply:
Yep. I will take care in next refactoring functional tests.